components/advocates/AdvocatesJoinSection.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <section class="join-section">
+    <div class="join-section__access">
+      <p class="join-section__access-description">
+        The Qiskit advocate program is a global program that provides support to the individuals who actively contribute to the Qiskit Community.
+      </p>
+      <AppCtaBtn
+        class="join-section__access-cta"
+        v-bind="joinAction"
+      />
+    </div>
+    <CompactFeature
+      v-for="benefit in advocateBenefits"
+      :key="`benefit-${benefit.icon}`"
+      class="join-section__feature"
+      v-bind="benefit"
+    />
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+import CompactFeature from '~/components/ui/CompactFeature.vue'
+import AppCtaBtn from '~/components/metal/AppCtaBtn.vue'
+
+type Benefit = Pick<CompactFeature, 'icon'|'title'|'description'>
+
+@Component({ components: { CompactFeature, AppCtaBtn } })
+export default class extends Vue {
+  advocateBenefits: Array<Benefit> = [
+    {
+      icon: 'high-five.svg',
+      title: 'Network with experts and enthusiasts',
+      description: 'Advocates will be added to a group of quantum experts and will be a part of regular information sharing sessions.'
+    },
+    {
+      icon: 'global-network.svg',
+      title: 'Access to Qiskit core members and projects',
+      description: 'Advocates will receive special access to core members of the Qiskit team for questions and brainstorming ideas.'
+    },
+    {
+      icon: 'location.svg',
+      title: 'Invitation to events',
+      description: 'Active Qiskit Advocates will be invited to attend global events created for the quantum computing community.'
+    }
+  ]
+
+  joinAction = {
+    url: '#meet-the-advocates',
+    label: 'Find an advocate'
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+  @import '~carbon-components/scss/globals/scss/typography';
+  .join-section {
+    @include contained();
```

Comment:
In very large, padding is not even on both sides, I think contained is the problem. If you change the `__feature` margin, this should have `padding: 0 $spacing-05;`  to match the design.

Reply:
@korgan00 - I think I'm missing what this looks like, can you please attach a screenshot :pray:

Reply:
![2020-11-09 10_08_23-Window](https://user-images.githubusercontent.com/4138279/98521710-16436b80-2274-11eb-8e81-9ac007b1c0a5.png)

Reply:
@korgan00 - I know that this `contained()` mixin was created long before I began on this project, and it seems like it's supposed to be that way?

@y4izus - can you confirm?

Reply:
You both are right. On the one hand, yes, we use the `contained()`, but on the other hand, if I have understood Goyo properly, this `contained()` is making your code more complicated when managing the margins/paddings. If you add the `padding: 0 $spacing-05;` right before the `contained()`, you can easily align the items.

@korgan00 , that's what you were trying to explain, right?

Reply:
If I'm understanding @korgan00 correctly... the design shows `1rem` of left and right padding, but here, our container is showing `padding-left: 2.5rem; padding-right: 1rem`... and this is the problem, is that right?

The way I'm using the `contained` mixin on this page, I believe (correct me if I'm mistaken), is consistent with how it's used on other pages.

Reply:
The design shows 2rem on each side. In your implementation, you have 1.5 from the container padding and 1 from the margin on the right side, and 2.5 from the container on the left.
That forces to make a difference between `__feature`  and `__access` margins and makes the CSS more complex.
If you apply this little change. You can merge the margins, flex, and media-queries from the `__feature`  and `__access`
```suggestion
    @include contained();
    padding: 0 1rem;
```